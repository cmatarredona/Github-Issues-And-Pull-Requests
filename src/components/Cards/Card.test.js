import { render } from "@testing-library/react";
import Card from "./Card";
it("Render the 'Card' component", () => {
  render(<Card>Hello world!</Card>);
  expect("Hello world!").toMatchSnapshot();
});

