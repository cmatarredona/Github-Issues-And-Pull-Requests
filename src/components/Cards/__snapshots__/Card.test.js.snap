// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the 'Card' component 1`] = `"Hello world!"`;
